src/bitcoin-util.cpp

```diff
@@ -60,7 +60,10 @@ static int AppInitUtil(ArgsManager& args, int argc, char* argv[])
             strUsage += FormatParagraph(LicenseInfo());
         } else {
             strUsage += "\n"
-                "Usage:  bitcoin-util [options] [commands]  Do stuff\n";
+                "The bitcoin-util tool provides bitcoin related functionality that does not rely on the ability to access a running node. Available [commands] are listed below.\n"
+                "\n"
+                "Usage:  bitcoin-util [options] [command]\n"
+                "or:     bitcoin-util [options] grind <hex-block-header>\n";
```

Comment:
What about, instead of this, we just remove it altogether in favor of the `Commands` section that's already there? That way, it will follow the same pattern as `bitcoin-wallet`.

Reply:
I think it's helpful to detail the only command.